src/components/Asset/Edit/FormEditMetadata.tsx

```diff
@@ -65,7 +66,7 @@ export default function FormEditMetadata({
       getFileInfo(asset.metadata.links[0], providerUrl, 'url').then(
         (checkedFile) => {
           // set valid false if url is using google drive
-          if (asset.metadata.links[0].includes('google.com')) {
```

Comment:
not sure we want to prevent sample files as well, those would work pretty well with google links

Reply:
Interesting, at the moment we are blocking `drive.google` links. Shall I just remove that validation altogether?

Reply:
Not sure if this is required. while it's been discussed, I'll approved this PR as solved what described in issue.